app/controllers/events/view/tickets/index.js

```diff
@@ -0,0 +1,14 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default Controller.extend({
+  totalOrders: computed('model.orderStats.orders.completed', 'model.orderStats.orders.placed', function() {
+    return this.model.orderStats.orders.completed + this.model.orderStats.orders.placed;
```

Comment:
are you sure they are numerical values and not strings?

Reply:
@CosmicCoder96 Since the corresponding server scheme sends back data in this format:

```python
result = {
            'total': total or 0,
            'draft': draft or 0,
            'cancelled': cancelled or 0,
            'pending': pending or 0,
            'expired': expired or 0,
            'placed': placed or 0,
            'completed': completed or 0
        }
```
I think it'll be safe to assume that they are numbers